client/src/pages/guide/english/css/forms/index.md

```diff
@@ -16,8 +16,51 @@ Form elements are different types of input elements, like text fields, checkboxe
 
 Inside the form tags there are also some important attribute which are:
 -action : the page that the data will sent to
--method : type of request which are: 'GET' and 'POST'
+-method : type of request which are: 'GET' and 'POST'.
 
+-GET: 
+1.Appends form-data into the URL in name/value pairs , so never use GET to send sensitive data.
+2.The length of a URL is limited (about 3000 characters).
```

Comment:
these aren't formatted properly for markdown, so it all ends up on the same line. Similarly below. Please convert all the remaining markup to appropriate markdown and add spacing as needed.